src/lib/Layout/SideBarLayout/SideBarLayout.tsx

```diff
@@ -11,19 +11,21 @@ import { useSideBarLayoutContext } from "./SideBarLayoutContext";
 interface SideBarLayoutProps extends PropsWithChildren {
   brand?: ReactNode;
   footer?: ReactNode;
+  theme?: "dark" | "light";
 }
 
 const SideBarLayout = (props: SideBarLayoutProps) => {
-  const { brand, children, footer } = props;
+  const { brand, children, footer, theme } = props;
 
   const [isOpen, setIsOpen] = useState(true);
   const toggleSidebar = () => setIsOpen((prev) => !prev);
 
-  const { brand: contextBrand, userDropDownMenu, userDropDownMenuToggle } = useSideBarLayoutContext();
+  const { brand: contextBrand, theme: contextTheme, userDropDownMenu, userDropDownMenuToggle } = useSideBarLayoutContext();
+  const chosenTheme = theme ?? contextTheme ?? "dark";
```

Comment:
With my changes mentioned in the other files you can now do
```suggestion
  const chosenTheme = theme ?? contextTheme;
```

here because contextTheme for sure has a value now :-)

Reply:
done